static/templates/single_message.handlebars

```diff
@@ -17,6 +17,8 @@
                                     <span class="sender_name-in-status auto-select sender_info_hover">{{msg/sender_full_name}}</span>
                                     {{#if sender_is_bot}}
                                     <i class="zulip-icon bot" aria-hidden="true"></i>
+                                    {{else if sender_is_realm_admin}}
+                                    <i class="zulip-icon crown" aria-hidden="true"></i>
```

Comment:
`<i class="zulip-icon crown" aria-hidden="true" title="Admin"></i>` for hover behavior.

Reply:
If you plan on implementing this, it'd make sense to add a title attribute to the bot icon also for consistency